### PR TITLE
Patch issue 287

### LINKED ISSUE #287: [all] java.lang.OutOfMemoryError: Metaspace   ---->WebService
Hello @jarlah , I'm doing tests with the webservice and today I wake up with this error:
error_webservice

Do you know if there is a fix to increase the metaspace for this module?

### PR DESCRIPTION
**Pull request by pingwiniasty**
Incremets memory limits in default config.
Touches only senstive services 

Fixes #287

### CODE REVIEW

File: core/src/main/resources/application.ini [hunk 1]
```diff
@@ -1,6 +1,6 @@
--J-Xms130m
+-J-Xms128m
```
Comment (jarlah) [inline core/src/main/resources/application.ini]: this doesnt seem right. You are going from 130m to 128m. same for the other two similar Xms below

Comment (jarlah) [reply core/src/main/resources/application.ini]: or are you just saying hey lets use computer rounded numbers?

Comment (pingwiniasty) [reply core/src/main/resources/application.ini]: You are right, I wan to 2^n numbers :D

Comment (jarlah) [reply core/src/main/resources/application.ini]: xD 
No problems ;)

**Pull request merged**